dynamic.tex

```diff
@@ -201,8 +201,9 @@ \section{JSX}\label{s:dynamic-jsx}
 and \texttt{for} loops don't return anything.
 (We could use a loop to build up a string through repeated concatenation,
 but \texttt{map} is cleaner.)
-And note: we must return exactly one node,
-because this is one function call.
+And note: since we are calling the function \texttt{ReactDOM.render} once,
+we must return one large DOM node
+(and cannot return a bunch of separate <li> nodes).
```

Comment:
We'll need `\texttt{{\textless}li{\textgreater}}` or `\htmltag{li}` (prefer the latter.